codegen/src/main/scala/akka/grpc/gen/Main.scala

```diff
@@ -26,8 +26,8 @@ object Main extends App {
 
   val req = CodeGeneratorRequest.parseFrom(inBytes)
   val out =
-    if (req.getParameter.toLowerCase.contains("language=scala")) ScalaServerCodeGenerator.run(req)
-    else JavaServerCodeGenerator.run(req)
+    if (req.getParameter.toLowerCase.contains("language=scala")) ScalaBothCodeGenerator.run(req)
+    else JavaBothCodeGenerator.run(req)
```

Comment:
I guess we want to make this configurable - ticketify?

Reply:
https://github.com/akka/akka-grpc/issues/155